wiki/Game_modes/en.md

```diff
@@ -0,0 +1,50 @@
+<!-- TODO: 
+- retake images to be ASC-compliant (update with respective game mode stubs)
+- add image for selecting game mode through Mode button -->
+
+# Game modes
+
+In osu!, a **game mode** is one of the four distinct configurations included in the osu! client (osu!, osu!taiko, osu!catch, and osu!mania). All of which are based on, or inspired by, other existing rhythm video games. 
+
+*Note: Each game mode has it's own icon/symbol: ![][o!s] ![][o!t] ![][o!c] ![][o!m] (osu!, osu!taiko, osu!catch, and osu!mania respectively).*
+
+The default game mode is osu! (or, as it is unofficially referred to as,"osu!standard"). Although osu! will open with the game mode that the last signed in player last played. 
+
+Players can change game modes by either clicking on the `Mode` button near the bottom-right of their screen, or by pressing `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, or `Ctrl`+`4` to switch to osu!, osu!taiko, osu!catch, or osu!mania respectively.
+
+## ![][o!s] osu!
+
+*Main page: [osu!](osu!)*
+
+*osu!* (a.k.a. *osu!standard*) is the default game mode for the osu! client, and is based on the original [iNiS](https://en.wikipedia.org/wiki/INiS "Wikipedia") rhythm video game [Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. Players here are expected to tap, slide, and spin to the beat of songs.
+
+![Gameplay of osu!](/wiki/shared/Interface_osu.jpg "osu! gameplay")
```

Comment:
might be better to include images right below headers?

Reply:
Was thinking abt that. My only concern would be that the size of the image makes the text too separated from the header